backend/src/main/scala/cromwell/backend/BackendLifecycleActorFactory.scala

```diff
@@ -48,7 +44,5 @@ trait BackendLifecycleActorFactory {
   lazy val fileHashingFunction: Option[FileHashingFunction] = None
   lazy val fileHashingWorkerCount: Int = 50
 
-  final lazy val fileContentsHasherActor: ActorRef = actorSystem.actorOf(RoundRobinPool(fileHashingWorkerCount)
-    .props(FileHasherWorkerActor.props(fileHashingFunction)),
-    "FileContentsHasherActor")
+  def fileContentsHasherActorProps: Props = FileHasherWorkerActor.props(fileHashingFunction)
```

Comment:
An alternative scheme I thought about was passing in a `TokenDistributingActor` to the `FileHashingActor` as part of the props. The idea being, you can't start working until you get a `WorkToken` and you'd return in when you're done.

I moved away from that because the built-in `Router`s just seemed like a neater fit.